lib/autoload/polyglot/language/python.rb

```diff
@@ -0,0 +1,25 @@
+class Polyglot::Language::Python < Polyglot::Language
+  class Python2Point7 < Polyglot::Language::Python
+    class << self
+      def instance
+        root
```

Comment:
I imagine the list of classes for the forseeable future be:
 - Python
   - *Python 2.7*
   - *Python 3.4*
 - *JavaScript*
   - JediScript (CS1101S)
 - *Java*
 - *C*
 - *C++*

So the root of every language is the most descended class which is implemented in code, these are in italics. Common parent classes (eg Python) share the same syntax highlighter. Subclasses can have a different evaluator...

So you can see why this is complex.

I'm not just asking for root, I'm asking for names in general :(